VEuPathDB/web-monorepo
# Multi blast 3

### Description

This PR reverts the multi-blast package to version 3, which works with what is on qa sites.

### Testing

1. Checkout this branch
2. Create `packages/sites/genomics-site/.env` if you don't already have one (see example below)
3. Run `yarn install`
4. Run `yarn nx start @veupathdb/genomics-site`

#### Example `.env`

```
VEUPATHDB_LOGIN_USER=****
VEUPATHDB_LOGIN_PASS=****

BASE_PROXY_URL=https://qa.plasmodb.org
LEGACY_WEB_APP_URL=${BASE_PROXY_URL}/plasmo
PROJECT_ID=PlasmoDB

CGI_BIN_ENDPOINT=/cgi-bin
CGI_BIN_URL=${BASE_PROXY_URL}/cgi-bin

WDK_SERVICE_ENDPOINT=/wdk-service
WDK_SERVICE_URL=${LEGACY_WEB_APP_URL}/service

EDA_ENABLED=false
EDA_SERVICE_ENDPOINT=/eda
EDA_SERVICE_URL=${BASE_PROXY_URL}/eda
EDA_EXAMPLE_ANALYSES_AUTHOR=0
EDA_SINGLE_APP_MODE=pass

MULTI_BLAST_ENDPOINT=/multi-blast
MULTI_BLAST_URL=${BASE_PROXY_URL}/multi-blast

SITE_SEARCH_SERVICE_ENDPOINT=/site-search
SITE_SEARCH_SERVICE_URL=${BASE_PROXY_URL}/site-search

USER_DATASETS_WORKSPACE_ENABLED=true
USER_DATASETS_WORKSPACE_IMPORT_SERVICE_ENDPOINT=/dataset-import
USER_DATASETS_WORKSPACE_IMPORT_SERVICE_URL=${BASE_PROXY_URL}/dataset-import

DOCUMENTS_ENDPOINT=/documents
DOCUMENTS_URL=${BASE_PROXY_URL}/documents

ASSETS_ENDPOINT=/assets
ASSETS_URL=${BASE_PROXY_URL}/assets

COMMUNITY_SITE=//qa.static-content.veupathdb.org/

FACEBOOK_URL=https://facebook.com/pages/EuPathDB/133123003429972
TWITTER_URL=https://twitter.com/VEuPathDB
REDDIT_URL=https://www.reddit.com/r/BRC_users/
YOUTUBE_URL=https://youtube.com/user/EuPathDB/playlists

ROOT_ELEMENT=#wdk-container
ROOT_URL=/a/app
LEGACY_WEB_APP_ENDPOINT=/a

BROWSERSLIST_ENV=modern

```
